apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerException.cs

```diff
@@ -74,7 +74,7 @@ public bool IsRetryable
         internal static bool TryTranslateRpcException(Exception possibleRpcException,
             out SpannerException spannerException)
         {
-            spannerException = possibleRpcException as SpannerException;
+            spannerException = null;
             var rpcException = possibleRpcException as RpcException;
```

Comment:
C# 7 will make a lot of this simpler, e.g. with pattern matching. If you fancy rebasing to master, you should be able to start using it whenever you want...